tables/output_tables.Rmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "Tables output for manuscript"
+author: "Run Jin"
+date: "10/27/2021"
+output:
+  pdf_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+
+library(tidyverse)
```

Comment:
A couple of dependencies questions:

- Is `openxlsx` on the Docker image already?
- Have you run this in the Docker image? Just want to make sure all the tidyverse functionality is supported using the versions we have on that image. This has been a source of some points of friction for this project 😅

Reply:
@jaclyn-taroni, thanks for reviewing this! Yes you are right - I did not run it in the Docker before... I modified the docker file but I saw that I cannot pull the new one until the PR is merged so I also did a install.packages("openxlsx") and ran it in docker, which worked just fine.

Let me know how to proceed next.

Reply:
If you were able to run it successfully using the current version of the image with `install.packages("openxlsx")` and CI passes, I don't think we need to take any additional steps.

Reply:
Thanks! I removed the extra line I added to Dockerfile before - so now there is no change to the Dockerfile.

Reply:
We still need the addition of `openxlsx` to the Dockerfile. Once this is merged, it will push the new image to Dockerhub.

Reply:
Oh my bad - added back now.